openapi.yml

```diff
@@ -0,0 +1,189 @@
+          schema:
+            type: integer
+          in: query
+        - name: date
+          description: Starting date for retrieving activities
+          schema:
+            format: date
+            type: string
+          in: query
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Activity"
+          description: An array of activities
+      summary: Get list of activities for a team
+  "/teams/{teamId}/activities/{activityId}":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+        - name: activityId
+          description: Numeric ID of the activity
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Activity"
+          description: A single activity detail
+      summary: Get a single activity detail
+  "/teams/{teamId}/members":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Member"
+          description: An array of team members
+      summary: Get list of team members
+  "/teams/{teamId}/members/{memberId}":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+        - name: memberId
+          description: Numeric ID of the member
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Member"
+          description: A single team member
+      summary: Get a single team member
+  "/teams/{teamId}/notes":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Note"
+          description: An array of notes
+      summary: Get list of notes for the team
+components:
+  schemas:
+    Team:
+      type: object
+      properties:

```

Comment:
The properties should correspond to the following data structure

```
    {
        "id": 6237, 
        "name": "frenke", 
        "primary_color": "", 
        "secondary_color": "",
        "role": 1
    }, 
    {
        "id": 7369, 
        "name": "Hurricanes", 
        "primary_color": "#ed2637", 
        "secondary_color": "#ffffff",
        "role": 2
    }
```

Reply:
done in latest commit